### PR TITLE
Feature/update nonprofit page

### PR DESCRIPTION
**Pull request by QuincyLarson**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

I updated the nonprofit project landing page based on a design by @CodeNonprofit and @CarlJKashnier. I also created a route to a list of nonprofit resources, which we agreed looks better as a formal page on Free Code Camp (as opposed to a wiki article).

### CODE REVIEW

File: client/less/main.less [hunk 1]
```diff
@@ -1179,4 +1179,8 @@ and (max-width : 400px)  {
 @import "jobs.less";
 @import "challenge.less";
 @import "toastr.less";
-@import "map.less";
\ No newline at end of file
+@import "map.less";
+
+ol, li {
```
Comment (raisedadead) [inline client/less/main.less]: This will render any and all `ol` and `li` on all pages that use this including the challenges.

**Comment by raisedadead** [comment]
Also just out of curiosity I didn't see the new route being linked anywhere on the non-profits page, not sure if that is expected?

File: client/less/main.less (hunk 1 above)
Comment (QuincyLarson) [inline client/less/main.less]: @raisesadead yes - this actually is the usual desired behavior. Somehow our ol elements haven't had bullets, which isn't desired. This remedies that.

Comment (BerkeleyTrue) [inline client/less/main.less]: This should really come before the import of the specific imports.

Comment (QuincyLarson) [inline client/less/main.less]: @raisedadead that's a good point. we are linking nonprofits to this through other means.

**Pull request merged**